pkg/oci/utils_test.go

```diff
@@ -80,6 +80,27 @@ func TestMinimalPodConfig(t *testing.T) {
 		Console:             consolePath,
 	}
 
+	expectedMounts := []vc.Mount{
+		{
+			Source:      "proc",
+			Destination: "/proc",
+			Type:        "proc",
+			Options:     nil,
+		},
+		{
```

Comment:
@mcastelino  These are just the tests, but I am currently doing the bind-mounts in the same namespace as host. Need to add the change to enter a separate mount namespace for 3.0